### ISSUE TITLE
FEAT close task button

### ISSUE THREAD

**Issue by Jemoka** [Shabang-Systems/Condution#240]
**Is your feature request related to a problem? Please describe.**
> #suggestion a checkbox or button to hit when task creation is done. Currently I have to tap at a white space below my tasks to get out of it. (Could also be user error)

**Describe the solution you'd like**
A close task button.

**Additional context**
base 1.1.1

**Comment by Nidwan** [comment]
Yes, it really surprised me first time I used the app. I often tap another task by mistake trying to close an open task.
